build.sbt

```diff
@@ -415,6 +415,7 @@ lazy val docs = project("docs")
       10.seconds
     },
     paradoxGroups := Map("Language" -> Seq("Scala", "Java")),
+    projectInfoVersion := (if (isSnapshot.value) "snapshot" else version.value),
```

Comment:
This should be placed in `inThisBuild` so that it applies (and sets) every sbt module version correctly. Around https://github.com/apache/incubator-pekko-http/blob/9e8e1e79cf27eca794220eb8d53152e30aebec1f/build.sbt#L45 is the best spot.